### DISCUSSION TITLE
Dark mode resets to light mode when using back button

### DISCUSSION THREAD

**Discussion by danieltj27** [community/community#19994] [Projects and Issues]
I'm currently using Dark Mode and I was looking at my profile [here](https://github.com/danieltj27).

I click the plus icon (+) and clicked on _Create Organisation_.

It took me to the pricing page for organisations which is defaulted to Light Mode (or day mode or whatever).

I click the back button in my browser and it took me back to my profile in Light Mode, **_not_** Dark Mode.

Very insignificant bug, but still definitely a bug nonetheless.

For what's it's worth; navigating to another page or refreshing it will revert back to Dark Mode so it's not a permanent issue. Just quite jarring to see it.

**Comment by 0xdsm** [comment]
You've the same problem as me, probably a CSS or JS loading error. To fix that we need to report direct on the GitHub Support page:
https://support.github.com/contact/bug-report
